components/ArticleComments.jsx

```diff
@@ -0,0 +1,29 @@
+import { getArticleComments } from "../api";
+import { useEffect, useState } from "react";
+import CommentCard from "./CommentCard";
+
+export default function ArticleComments({article_id}) {
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const loadingMessage = 'Comments loading...';
+
+    useEffect(() => {
+        getArticleComments(article_id)
+            .then(comments => {
+            setComments(comments)
+            setIsLoading(false)
+        })
+        .catch((error) => {
+            setIsLoading(false)
+            console.log(error)
+        })
+    }, []);
+    
+    return (isLoading ? loadingMessage : 
+        <>
```

Comment:
We do have some articles with no associated comments. In this case it would be nice to conditionally render a message alluding to this